### ISSUE TITLE
PopupButton does not render grid correctly

### ISSUE THREAD

**Issue by Dudeplayz** [mvysny/karibu-dsl#33]
I'm trying to show a grid with some information on clicking a PopupButton. But the grid items are empty. The header is rendered correctly. With ListBox it works fine. Maybe it's a vaadin related problem.
Tested with: `vaadin: 14.4.5` and `karibu-dsl: 1.0.6`

```kotlin
    private fun buildProviderPopup(): PopupButton {
        val popup = PopupButton("Providers")
        val grid = Grid<String>()
        grid.setItems(listOf("test", "test2"))
        grid.addColumn { it }.setHeader("Header").width = "20em"
        grid.width = "30em"
        popup.setPopupContent(grid)
        return popup
    }
```

![image](https://user-images.githubusercontent.com/15174076/117974517-ef064d80-b32d-11eb-8213-c02d5c411cc3.png)

**Comment by mvysny** [comment]
Hmm, that looks like a Vaadin bug to me. Could you please try out the newest Vaadin 14.6.0 whether the issue persists? If yes, I'd suggest to open a bug report at https://github.com/vaadin/flow-components/issues

**Comment by Dudeplayz** [comment]
You are right. I have done some testing.
It was fixed in vaadin version `14.4.6`. 
Also tested: 
- `14.4.4` broken
- `14.4.5` broken
- `14.4.6` fixed
- `14.4.9` still fixed
- `14.4.10` still fixed
- `14.5.5` still fixed
